### PR TITLE
Changes login link to authorize

### PR DESCRIPTION
**Pull request by danielzeller-okta**
Very small change to the links object docs.

@karlmcguinness-okta @mattloughran-okta @slee-okta @joelfranusic-okta @eklein-okta

### CODE REVIEW

File: _source/docs/api/resources/idps.md [hunk 1]
```diff
@@ -1149,16 +1149,16 @@ Webhook authorization settings for an IdP provisioning or account link transacti
...
-| acs                | SAML 2.0 Assertion Consumer Service URL for Okta SP                                                                      |
-| users              | IdP Users                                                                                                                |
-|--------------------+--------------------------------------------------------------------------------------------------------------------------|
+|--------------------+------------------------------------------------------------------------------------------------------------------------|
+| Link Relation Type | Description                                                                                                            |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------- |
+| self               | The primary URL for the IdP                                                                                            |
+| authorize          | The URL to authenticate user with [OAuth 2.0 Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-4.1) |
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/idps.md]: [OAuth 2.0 authorization endpoint](https://tools.ietf.org/html/rfc6749#section-3.1) for the IdP

**Comment by karlmcguinness-okta** [comment]
Reviewed with minor comment

**Comment by danielzeller-okta** [comment]
I've made the change @karlmcguinness-okta

**Comment by tthompson-okta** [comment]
:ship:

**Pull request merged**